src/Orleans/Streams/PubSub/PubSubSubscriptionState.cs

```diff
@@ -45,6 +46,7 @@ internal enum SubscriptionStates
         {
             SubscriptionId = subscriptionId;
             Stream = streamId;
+            consumerExtension = streamConsumer;
             consumerReference = streamConsumer as GrainReference;
             state = SubscriptionStates.Active;
```

Comment:
@ReubenBond  hey Reuben, can changes in this file affect persistence? as in, when serializer serialize entity from persistence storage, the serialization would be problematic before and after this change?